tests/test-pull-corruption.sh

```diff
@@ -55,7 +55,16 @@ do_corrupt_pull_test() {
 
 # FIXME - ignore errors here since gjs in RHEL7 has the final
 # unrooting bug
-gjs $(dirname $0)/corrupt-repo-ref.js ${repopath} main || true
+
+# super hacky fix to support both installed and uninstalled tests. we'll
+# probably want to just enable it for one of them.
+if [ -f /lib64/libostree-1.so ]; then
```

Comment:
Also please note that this "super hacky fix" is going to fail on 32-bit, on Debian and its derivatives (it's `/usr/lib/x86_64/libostree-1.so` there), under jhbuild and so on. See #232 for the use of `LD_LIBRARY_PATH` that fixes this differently.